### PR TITLE
April news

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

File: content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,655 @@
...
+
+### 18.000 packages tracked on R-universe!
+
+Currently r-universe is tracking and indexing of over 18.000 R packages! These are a mix of packages found on popular networks like [CRAN](https://cran.r-project.org/) or [Bioconductor](https://www.bioconductor.org/), and packages that were registered by users.
+
+In previous posts we already explained [how to create your personal CRAN-like repository](/blog/2021/06/22/setup-runiverse/) and publish packages on r-universe yourself. A new post explains the other part: how the scraper [automatically finds packages on CRAN and Bioconductor](/blog/2023/04/03/cran-to-git/) that should be included in r-universe.
+
+All in all, R-universe provides a great way to [discover and explore many packages](/blog/2023/02/27/runiverse-discovering/).
+
+
```
Comment (yabellini) [inline content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd:69]: ## Help us put together the rOpenSci Community Call calendar for the year!

After a break, we have resumed our community calls.  🎉  
And we want to know what topics **you** would like us to cover. 

We hold [Community Calls](https://ropensci.org/commcalls/) every 3 months to share knowledge that is relevant to our community and consistent with [our vision and mission](https://ropensci.org/about/). These are free and open for anyone to attend and provide opportunities for us to connect with rOpenSci community members around the world.

We’ve [opened an issue in this repository for each topic we’re considering](https://github.com/ropensci-org/community-calls/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc). We'd like **your input and "votes"** on these, and **your suggestions for other topics** we haven't though of.

We will also be very grateful if you propose speakers, resources, or encourage others to weigh in by sharing the link to your favorite topic.

You can [read more details on our community calls repository](https://github.com/ropensci-org/community-calls). 

We look forward to hearing your ideas.  Let's build the calendar together.

Comment (yabellini) [inline content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd:69]:
```suggestion
## Help us put together the rOpenSci Community Call calendar for the year!

After a break, we have resumed our community calls.  🎉  
And we want to know what topics **you** would like us to cover. 

We hold [Community Calls](https://ropensci.org/commcalls/) every 3 months to share knowledge that is relevant to our community and consistent with [our vision and mission](https://ropensci.org/about/). These are free and open for anyone to attend and provide opportunities for us to connect with rOpenSci community members around the world.

We’ve [opened an issue in this repository for each topic we’re considering](https://github.com/ropensci-org/community-calls/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc). We'd like **your input and "votes"** on these, and **your suggestions for other topics** we haven't though of.

We will also be very grateful if you propose speakers, resources, or encourage others to weigh in by sharing the link to your favorite topic.

You can [read more details on our community calls repository](https://github.com/ropensci-org/community-calls). 

We look forward to hearing your ideas.  Let's build the calendar together.     
```

File: content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,669 @@
...
+You can read this post [on our blog](`r url`).
+Now let's dive into the activity at and around rOpenSci!
+
+## rOpenSci HQ
+
+### Meeting the stars of the R-universe: Athanasia Monika Mowinckel
+
+Knowing our community's stories helps us to learn about the people behind our software, brings us closer and offers us new opportunities. To share some of these community stories, we created the rOpenSci interview series [_"Meeting the stars of the R-Universe"_](/tags/r-universe-stars/).
+
+Working with the human brain requires specific software and complex developments, however R appears as the natural way to collect hundreds of data that need to be analyzed and the R-Universe as the most friendly place to share and invite to collaborate. Our third stop bring us to Norway to talk with [Athanasia Monika Mowinckel](/blog/2023/03/30/r-universe-stars-3-en/).
```
Comment (steffilazerte) [inline content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd:48]:
```suggestion
Working with the human brain requires specific software and complex developments. However R appears to be the natural way to collect and analyze the huge amount of data needed, and the R-Universe the most friendly place to share and invite collaborations. Our third stop brings us to Norway to talk with [Athanasia Monika Mowinckel](/blog/2023/03/30/r-universe-stars-3-en/).
```

File: content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd [hunk 3]
```diff
@@ -0,0 +1,669 @@
...
+### Multilingual posts
+
+Have you noticed several of our recent blog posts were translated to Spanish, and in one case, French?
+We've started adding actual multilingual infrastructure to our website.
+You can find all posts in Spanish in <https://ropensci.org/es/archive> (and the lone French post in <https://ropensci.org/fr/archive>).
+From any post that has a translation, the sidebar on the right (or... at the bottom if you're reading on mobile!) has a link to translations of the post, see this [example](/blog/2023/02/28/r-universe-stars-2-en/).
+
+We plan to improve the multilingualism of our website over time.
+
+### 18.000 packages tracked on R-universe!
```
Comment (steffilazerte) [inline content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd:61]: Does this number need to be updated?
```suggestion
### 18,000 packages tracked on R-Universe!
```

File: content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd [hunk 4]
```diff
@@ -0,0 +1,669 @@
...
+
+Currently r-universe is tracking and indexing of over 18.000 R packages! These are a mix of packages found on popular networks like [CRAN](https://cran.r-project.org/) or [Bioconductor](https://www.bioconductor.org/), and packages that were registered by users.
```
Comment (steffilazerte) [inline content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd:63]:
```suggestion
Currently R-Universe is tracking and indexing over 18,000 R packages! These are a mix of packages found on popular networks like [CRAN](https://cran.r-project.org/) or [Bioconductor](https://www.bioconductor.org/), and packages that were registered by users.
```

File: content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd [hunk 5]
```diff
@@ -0,0 +1,669 @@
...
+
+In previous posts we already explained [how to create your personal CRAN-like repository](/blog/2021/06/22/setup-runiverse/) and publish packages on r-universe yourself. A new post explains the other part: how the scraper [automatically finds packages on CRAN and Bioconductor](/blog/2023/04/03/cran-to-git/) that should be included in r-universe.
```
Comment (steffilazerte) [inline content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd:65]:
```suggestion
In previous posts we explained [how to create your personal CRAN-like repository](/blog/2021/06/22/setup-runiverse/) and publish packages on R-Universe yourself. A new post explains the other part: how the scraper [automatically finds packages on CRAN and Bioconductor](/blog/2023/04/03/cran-to-git/) to include in the R-Universe.
```

File: content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd [hunk 6]
```diff
@@ -0,0 +1,669 @@
...
+
+All in all, R-universe provides a great way to [discover and explore many packages](/blog/2023/02/27/runiverse-discovering/).
```
Comment (steffilazerte) [inline content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd:67]:
```suggestion
All in all, R-Universe provides a great way to [discover and explore many packages](/blog/2023/02/27/runiverse-discovering/)!
```

File: content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd [hunk 7]
```diff
@@ -0,0 +1,669 @@
...
+
+## Help us put together the rOpenSci Community Call calendar for the year!
+
+After a break, we have resumed our community calls.  🎉  
+And we want to know what topics **you** would like us to cover. 
+
```
Comment (steffilazerte) [inline content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd:73]:
```suggestion
We've had [some](https://ropensci.org/commcalls/oct2022-champions/) [great](https://ropensci.org/commcalls/jan2023-targets/) community calls lately,  🎉  
and now we'd like to know what topics **you** would like us to cover in the future!

```

File: content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd [hunk 8]
```diff
@@ -0,0 +1,669 @@
...
+We hold [Community Calls](/commcalls/) every 3 months to share knowledge that is relevant to our community and consistent with [our vision and mission](/about/). These are free and open for anyone to attend and provide opportunities for us to connect with rOpenSci community members around the world.
```
Comment (steffilazerte) [inline content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd:74]:
```suggestion
We hold [Community Calls](/commcalls/) about every 3 months to share knowledge that is relevant to our community and consistent with [our vision and mission](/about/). These are free and open for anyone to attend and provide opportunities for us to connect with rOpenSci community members around the world.
```

File: content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd [hunk 9]
```diff
@@ -0,0 +1,669 @@
...
+
+We’ve [opened an issue in this repository for each topic we’re considering](https://github.com/ropensci-org/community-calls/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc). We'd like **your input and "votes"** on these, and **your suggestions for other topics** we haven't though of.
```
Comment (steffilazerte) [inline content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd:76]:
```suggestion
We’ve [opened an issue in this repository for each topic we’re considering](https://github.com/ropensci-org/community-calls/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc). We'd like **your input and "votes"** on these, and **your suggestions for other topics** we haven't thought of.
```

File: content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd [hunk 10]
```diff
@@ -0,0 +1,669 @@
...
+
+We will also be very grateful if you suggest speakers, resources, or encourage others to weigh in by sharing the link to your favorite topic.
+
+You can [read more details on our community calls repository](https://github.com/ropensci-org/community-calls). 
+
+We look forward to hearing your ideas. Let's build the schedule together.
```
Comment (steffilazerte) [inline content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd:82]:
```suggestion
We look forward to hearing your ideas. Let's build the schedule together!    
```

File: content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd [hunk 11]
```diff
@@ -0,0 +1,669 @@
...
+    
+And remember, you can always cowork independently on work related to R, work on packages that tend to be neglected, or work on what ever you need to get done!
+
+* Tuesday, May 2nd, 9:00 Americas Pacific / 16:00 UTC ["Spring Cleaning for R packages and scripts"](/events/coworking-2023-05/) *Hosted by [Steffi LaZerte](/author/steffi-lazerte/) and [Yanina Bellini Saibene](/author/yanina-bellini-saibene/)*
+    - Explore how other organizations keep their scripts/packages nice and clean
+    - Take a look at your R packages and scripts and give them a good spring cleaning\*
+    - Talk to our community host and other attendees and discuss tips for keeping on top of it all.
+    
+\* in the northern hemisphere at least, otherwise, give them a good *fall* cleaning!
+
```
Comment (steffilazerte) [inline content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd:100]: I'm just about to finalize the event posting

```suggestion

* Tuesday, June 6th, 9:00 Australia Western / 01:00 UTC ["Integrating and merging datasets from different sources"](/events/coworking-2023-06/) *Hosted by community host [Cynthia Huang](/author/cynthia-huang) and [Steffi LaZerte](/author/steffi-lazerte/)*
    - Spend time integrating datasets for your own work
    - Explore how others tackle the problem of merging datasets from different sources or different data versions
    - Talk to our community host and other attendees and discuss strategies for integrating datasets.
    
```

File: content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd [hunk 12]
```diff
@@ -0,0 +1,669 @@
...
+### 18.000 packages tracked on R-universe!
+
+Currently r-universe is tracking and indexing of over 18.000 R packages! These are a mix of packages found on popular networks like [CRAN](https://cran.r-project.org/) or [Bioconductor](https://www.bioconductor.org/), and packages that were registered by users.
+
+In previous posts we already explained [how to create your personal CRAN-like repository](/blog/2021/06/22/setup-runiverse/) and publish packages on r-universe yourself. A new post explains the other part: how the scraper [automatically finds packages on CRAN and Bioconductor](/blog/2023/04/03/cran-to-git/) that should be included in r-universe.
+
+All in all, R-universe provides a great way to [discover and explore many packages](/blog/2023/02/27/runiverse-discovering/).
+
+## Help us put together the rOpenSci Community Call calendar for the year!
```
Comment (steffilazerte) [inline content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd:69]:
```suggestion
### Help us put together the rOpenSci Community Call calendar for the year!
```

File: content/blog/2023-04-21-ropensci-news-digest-april-2023/index.md [hunk 13]
```diff
@@ -0,0 +1,233 @@
...
...his package are from: 1. Chang, E. J., Guerra, S. M., de Souza Penaloza, R. A. & Tabak, B. M. (2005) "Banking concentration: the Brazilian case". 2. Cobham, A. and A. Summer (2013). "Is It All About the Tails? The Palma Measure of Income Inequality". 3. Garcia Alba Idunate, P. (1994). "Un Indice de dominancia para el analisis de la estructura de los mercados". 4. Ginevicius, R. and S. Cirba (2009). "Additive measurement of market concentration" <doi:10.3846/1611-1699.2009.10.191-198>. 5. Herfindahl, O. C. (1950), "Concentration in the steel industry" (PhD thesis). 6. Hirschmann, A. O. (1945), "National power and structure of foreign trade". 7. Melnik, A., O. Shy, and R. Stenbacka (2008), "Assessing market dominance" <doi:10.1016/j.jebo.2008.03.010>. 8. Palma, J. G. (2006). "Globalizing Inequality: Centrifugal and Centripetal Forces at Work". 9. Shannon, C. E. (1948). "A Mathematical Theory of Communication". 10. Simpson, E. H. (1949). "Measurement of Diversity" <doi:10.1038/163688a0>.  It is available on [CRAN]( https://CRAN.R-project.org/package=concstats). It has been [reviewed](https://github.com/ropensci/software-review/issues/559) by Sebastian Wojcik, and Christopher T. Kenny.
```
Comment (steffilazerte) [inline content/blog/2023-04-21-ropensci-news-digest-april-2023/index.md:91]: Consider omitting the middle second half when you go to publish (doi links don't work as is and I don't think this much detail is required).
```suggestion
+ [concstats](https://docs.ropensci.org/concstats), developed by Andreas Schneider: Based on individual market shares of all participants in a market or space, the package offers a set of different structural and concentration measures frequently - and not so frequently - used in research and in practice. Measures can be calculated in groups or individually. The calculated measure or the resulting vector in table format should help practitioners make more informed decisions. It is available on [CRAN]( https://CRAN.R-project.org/package=concstats). It has been [reviewed](https://github.com/ropensci/software-review/issues/559) by Sebastian Wojcik, and Christopher T. Kenny.
```

File: content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd [hunk 14]
```diff
@@ -0,0 +1,669 @@
...
+
+### Caching the results of functions of your R package
+
+The R-hub blog has a post on [Caching the results of functions of your R package](https://blog.r-hub.io/2021/07/30/cache/) by Maëlle Salmon and Christophe Dervieux.
+It presents different approaches.
+A recent PR by Lukas Wallrich to rcrossref maintained by Najko Jahn illustrates the approach of [saving results to an environment](https://github.com/ropensci/rcrossref/pull/237) (and reminded us about the topic of caching!).
+
+### Deprecated data in a package with `delayedAssign()`
+
+If you need to change something like a function or function arguments in your package, you'll find many resources in say [rOpenSci dev guide](https://devguide.ropensci.org/evolution.html), or the [lifecycle package docs](https://lifecycle.r-lib.org/).
```
Comment (steffilazerte) [inline content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd:645]:
```suggestion
If you need to change something like a function or function arguments in your package, you'll find many resources in, say, the [rOpenSci dev guide](https://devguide.ropensci.org/evolution.html), or the [lifecycle package docs](https://lifecycle.r-lib.org/).
```

File: content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd [hunk 15]
```diff
@@ -0,0 +1,669 @@
...
+
+Explore [other use cases](/usecases) and [report your own](https://discuss.ropensci.org/c/usecases/10)! -->
+
+## Call for (co)maintainers
+
+### Call for maintainers
+
+If you're interested in maintaining any of the R packages below, you might enjoy reading our blog post [What Does It Mean to Maintain a Package?](/blog/2023/02/07/what-does-it-mean-to-maintain-a-package/) (or listening to its discussion on the [R Weekly highlights podcast](https://rweekly.fireside.fm/111) hosted by Eric Nantz and Mike Thomas)!
+
+- **[rvertnet](https://cran.r-project.org/web/packages/rvertnet/index.html)**, Retrieve, map and summarize data from the VertNet.org archives (<http://vertnet.org/>). Functions allow searching by many parameters, including taxonomic names, places, and dates. In addition, there is an interface for conducting spatially delimited searches, and another for requesting large datasets via email. [Issue for volunteering](https://github.com/ropensci-archive/rvertnet/issues/71).
```
Comment (steffilazerte) [inline content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd:617]:
```suggestion
- **[rvertnet](https://cran.r-project.org/web/packages/rvertnet/index.html)**, Retrieve, map and summarize data from the VertNet.org archives (<https://vertnet.org/>). Functions allow searching by many parameters, including taxonomic names, places, and dates. In addition, there is an interface for conducting spatially delimited searches, and another for requesting large datasets via email. [Issue for volunteering](https://github.com/ropensci-archive/rvertnet/issues/71).
```

File: content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd [hunk 16]
```diff
@@ -0,0 +1,669 @@
...
+
+- **[natserv](https://cran.r-project.org/web/packages/natserv/index.html)**. Interface to NatureServe (<https://www.natureserve.org/>). Includes methods to get data, image metadata, search taxonomic names, and make maps. [Issue for volunteering](https://github.com/ropensci-archive/natserv/issues/29).
+
+- **[geojsonlint](https://cran.r-project.org/web/packages/geojsonlint/index.html)**, Tools for linting GeoJSON. Includes tools for interacting with the online tool <http://geojsonlint.com>, the Javascript library geojsonhint (<https://www.npmjs.com/package/geojsonhint>), and validating against a GeoJSON schema via the Javascript library (<https://www.npmjs.com/package/is-my-json-valid>). Some tools work locally while others require an internet connection. [Issue for volunteering](https://github.com/ropensci-archive/geojsonlint/issues/22).
```
Comment (steffilazerte) [inline content/blog/2023-04-21-ropensci-news-digest-april-2023/index.Rmd:621]:
```suggestion
- **[geojsonlint](https://cran.r-project.org/web/packages/geojsonlint/index.html)**, Tools for linting GeoJSON. Includes tools for interacting with the online tool <https://geojsonlint.com>, the Javascript library geojsonhint (<https://www.npmjs.com/package/geojsonhint>), and validating against a GeoJSON schema via the Javascript library (<https://www.npmjs.com/package/is-my-json-valid>). Some tools work locally while others require an internet connection. [Issue for volunteering](https://github.com/ropensci-archive/geojsonlint/issues/22).
```

**Review by steffilazerte — COMMENTED** [review]
Looks good! Just a couple of suggestions and I added the coworking details. 

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case
* [X] publication date is ok
* [X] alternative text of images is informative
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)
* [X] YAML field 'preface' is present if necessary

**Comment by maelle** [comment]
@steffilazerte thanks so much!

**Pull request merged**